README.md

```diff
@@ -19,7 +19,7 @@
 <!-- @snipEnd -->
 
 ```ts @blockFrom:dev/test/examples/type.ts
-import { type } from "arktype"
+import { type } from "../../../src/main.js"
```

Comment:
This is a regression but I think it's just because I manually changed them recently 😂 Can you either manually change them back or fix the snip transform so they import from `arktype`?